src/components/user-profile.component.js

```diff
@@ -32,6 +33,9 @@ export const UserProfile = ({
       />
       <Text style={styles.title}>{user.name}</Text>
       <Text style={styles.subtitle}>{initialUser.login}</Text>
+      <Text style={styles.bio}>
+        {get(initialUser, ['bio', 'length'], 0) ? initialUser.bio : user.bio}
```

Comment:
So `initialUser` is the first user object that gets passed in as a prop when viewing a list of users (like the Followers screen for example). However that doesn't contain all the information needed for the user so another API call is made to get the full user data once the user profile screen is mounted. I pass `initialUser` in order to have the avatar and name show immediately while the call is still fetching.

Name probably is quite misleading - will definitely be more than happy to discuss renaming or restructuring this in a separate ticket <3